agenda/presentation/src/main/java/com/gkp/agenda/presentation/task/edittask/EditTaskScreen.kt

```diff
@@ -139,8 +373,22 @@ private fun EditTaskScreen(
 private fun TaskScreenPreview() {
     TaskyTheme {
         EditTaskScreen(
-            onClickCloseButton = {}
+            onClickEditCloseButton = {},
+            onClickTaskTitle = {},
+            onClickTaskDescription = {},
+            onClickEditSaveButton = {},
+            onClickDeleteButton = {},
+            onCLickReminderMenuItem = {},
+            onTaskDateSelected = {},
+            onTaskTimeSelected = { i: Int, i1: Int -> },
```

Comment:
Instead of all these lambdas in your `EditTaskScreen` that all share a commonality of interactions with your screen,  don’t you think there is a better way to encapsulate all the possible actions the user can perfom on the screen to replace all these lambdas with?

That way, you also benefit from not needing to update your usages of `EditTaskScreen` each time you add or remove interactions from your screen.